app/src/main/java/com/asfoundation/wallet/billing/share/BdsShareLinkRepository.kt

```diff
@@ -10,9 +10,10 @@ class BdsShareLinkRepository(private var api: BdsShareLinkApi) : ShareLinkReposi
   override fun getLink(domain: String, skuId: String?, message: String?,
                        walletAddress: String,
                        originalAmount: String?,
-                       originalCurrency: String?): Single<String> {
+                       originalCurrency: String?, paymentMethod: String): Single<String> {
```

Comment:
This could use a code style update.
There are lines with a singe arguments and others with multiple

Reply:
Agree, will change it manually for now